index.js

```diff
@@ -54,4 +59,37 @@ function updateMatchListElement() {
   matchListElement.appendChild(newMatchItem);
 }
 
+function addTeamsToTeamList({ teamA, teamB }) {
+  if (!teamList.find((element) => element.name === teamA)) {
+    teamList.push({ name: teamA, wins: 0 });
+  }
+
+  if (!teamList.find((element) => element.name === teamB)) {
+    teamList.push({ name: teamB, wins: 0 });
+  }
+}
+
+function updateRanksListElement() {
+  rankListElement.innerHTML = "";
+
+  teamList.forEach((team) => {
+    const newRankItem = document.createElement("li");
+    const itemText = document.createTextNode(
+      `${team.name} has ${team.wins} wins`
+    );
+    newRankItem.appendChild(itemText);
+    rankListElement.appendChild(newRankItem);
+  });
+}
+
+function updateTeamWins(team) {
+  const indexOfName = teamList.indexOf(
```

Comment:
```suggestion
  const indexOfWinningTeam= teamList.indexOf(
```
Toujours le maximum d'infos sur la variable.

Reply:
D'accord